base/applications/wusa/wusa.c

```diff
@@ -0,0 +1,19 @@
+/*
+* PROJECT:          ReactOS Setup
+* LICENSE:          GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:          Windows Update Stand-Alone installer.
+*                   Stub is required for some application at the installation phase.
+* COPYRIGHT:        Copyright 2018 Denis Malikov (filedem@gmail.com)
+*/
+
+#include "wusa.h"
+
+int
+WINAPI
+wWinMain(HINSTANCE hCurInst,
+          HINSTANCE hPrevInst,
```

Comment:
Extra space? (Usually those are either on one line, or aligned directly below each others, see https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&sr=1&s=wWinMain for examples)